ansible_collections/arista/cvp/molecule/cv_device_v3/test_cv_device_v3.yaml

```diff
@@ -0,0 +1,308 @@
+---
+- name: Test cv_device_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    CVP_DEVICES_MOVE_DEVICE:
+      - fqdn: leaf1
+        parentContainerName: "Leaf"
```

Comment:
Let's create a new container and then move it there.

Reply:
fixed